desktop/apps/categories3/components/GeneFamilyNav.js

```diff
@@ -41,20 +48,53 @@ const GeneFamilyLink = styled.a`
   }
 `
 
-const GeneFamilyNav = ({ geneFamilies }) => {
-  return (
-    <ResponsiveSidebar>
-      <GeneFamilyList>
-        {geneFamilies.map(geneFamily =>
-          <GeneFamilyItem key={geneFamily.id}>
-            <GeneFamilyLink href={`#${geneFamily.id}`}>
-              {geneFamily.name}
-            </GeneFamilyLink>
-          </GeneFamilyItem>
-        )}
-      </GeneFamilyList>
-    </ResponsiveSidebar>
-  )
+class GeneFamilyNav extends React.Component {
+  constructor (props) {
+    super(props)
+    this.handleClick = this.handleClick.bind(this)
+  }
+
+  handleClick (e) {
+    e.preventDefault()
+    const id = e.target.hash
+    const section = document.querySelector(id)
+    const topBuffer = 90
```

Comment:
Maybe this could be passed as a prop when we setting up the layout? If we decide to reshuffle the page seems strange that we would have to change the component.

Reply:
I didn't see the win in making it a prop, but your comment did encourage me to get rid of that repeated magic number and at least factor it out into a constant, so thanks!